### ISSUE TITLE
Trouble with video

### ISSUE THREAD

**Issue by ParkerRembe** [Montana-Media-Arts/341-web-design-Fall2020-resources#5]
How do we watch and star the Issues form

**Comment by haileymonaco** [comment]
If I did it right (which honestly who knows, I've never used this before) you should have three little tabs on the right of your screen that say "Watch" Star" and "Fork". If you click on the "Watch" you should have a drop down menu and select "watching" (I think??) and then for the "star" just click on it and it should bold the star which I believe means you've stared the page. 
Hope that helps!

**Comment by ParkerRembe** [comment]
Yes, that helped a lot. I didn't even notice those on the side of the page.

Thank You,
Parker
________________________________
